user-console/src/main/java/org/pentaho/mantle/client/admin/EmailAdminPanel.java

```diff
@@ -74,26 +89,53 @@ private Widget createEmailPanel() {
     vSpacer.setHeight( "20px" );
     mailPanel.add( vSpacer );
 
-    mailPanel.add( new Label( Messages.getString( "smtpHost" ) + ":" ) );
+    HorizontalPanel protocolHbox = new HorizontalPanel();
+    protocolHbox.add( new Label( Messages.getString( "connectionProtocol" ) + ":" ) );
+    protocolHbox.getElement().setId( "protocolPanel" );
+
+    SimplePanel hSpacer = new SimplePanel();
+    hSpacer.setWidth( "15px" );
+    protocolHbox.add( hSpacer );
+
+    protocolsListBox = new ListBox();
+    protocolsListBox.addItem( Messages.getString( "smtp" ), Messages.getString( "smtp" ) );
+    protocolsListBox.addItem( Messages.getString( "smtps" ), Messages.getString( "smtps" ) );
+    protocolsListBox.addItem( Messages.getString( "graphApiLabel" ), Messages.getString( "graphApiValue" ) );
```

Comment:
What is the need for localizing the _values_ (not the labels) of the listbox items?